### PR TITLE
[sil.brb.brao] update wordlist

### PR DESCRIPTION
**Pull request by MakaraSok**
A new wordlist has been obtained and updated.

### CODE REVIEW

File: release/sil/sil.brb.brao/LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-© 2021 SIL International
+© 2022 SIL International
```
Comment (darcywong00) [inline release/sil/sil.brb.brao/LICENSE.md:3]:
```suggestion
© 2021-2022 SIL International
```

File: release/sil/sil.brb.brao/README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Brao Lexical Model
 ===================
 
-© 2021 SIL International
+© 2022 SIL International
```
Comment (darcywong00) [inline release/sil/sil.brb.brao/README.md:4]:
```suggestion
© 2021-2022 SIL International
```

File: release/sil/sil.brb.brao/sil.brb.brao.kpj [hunk 3]
```diff
@@ -19,12 +19,12 @@
...
       <Filename>sil.brb.brao.model.kps</Filename>
       <Filepath>source\sil.brb.brao.model.kps</Filepath>
-      <FileVersion>1.0</FileVersion>
+      <FileVersion>1.1</FileVersion>
       <FileType>.kps</FileType>
       <Details>
         <Name>brao</Name>
-        <Copyright>© 2021 SIL International</Copyright>
-        <Version>1.0</Version>
+        <Copyright>© 2022 SIL International</Copyright>
```
Comment (darcywong00) [inline release/sil/sil.brb.brao/sil.brb.brao.kpj:26]:
```suggestion
        <Copyright>© 2021-2022 SIL International</Copyright>
```

**Comment by MakaraSok** [comment]
@darcywong00 thank you for catching that. 💯

**Comment by darcywong00** [comment]
I also ran a histogram of the wordlist. Confirming you intentionally wanted these characters

There's also parenthesis on this entry
`ចកាំង(អវ)`


| Count |  Codepoint | Name |
|----------|----------------|----------|
| 98	     | 0x200B      | Zero-width space |
| 4	     | 0x200D      | Zero-width joiner |
| 1         | 0xFEFF      | Zero-width no-break space |

**Comment by DavidLRowe** [comment]
@MakaraSok What is the status of this PR?

**Comment by DavidLRowe** [comment]
@MakaraSok Status?

**Comment by MakaraSok** [comment]
Hi David, Thank you for checking up on this. The last conversation I had with the wordlist author is that it would be published in September last year, but I have not yet heard back since.

I will follow up with that and see if we can close this issue. Many thanks!

**Comment by DavidLRowe** [comment]
It seems that this PR should be closed since the proposed changes have been made on PR #279

**Comment by DavidLRowe** [comment]
@mcdurdin Is this PR still needed? Should it be closed?

**Pull request closed without merging**